client/src/components/Comic.module.scss

```
@@ -0,0 +1,51 @@
+.comic {
+  padding: 12px;
+  text-align: center;
+  width: 180px;
+  height: 360px;
+
+  .comic:hover {
+    transform: scale(1.1);
+  }
+
+  &__cover {
+    max-height: 220px;
+    width: auto;
+    transition: all 0.2s ease-in-out;
```

It's better to only set a transition on the specific property you want to use it for, to avoid surprises later on.

```suggestion
    transition: transform 0.2s ease-in-out;
```